separate/runtimes/go/rootfs.sh

```diff
@@ -0,0 +1,12 @@
+cp /runtime/workload /bin/runtime-workload-elf
+cat > /bin/runtime-workload <<EOF
+#!/bin/sh
+/usr/bin/setup-eth0.sh
+/usr/bin/ioctl
+/usr/bin/factorial $((1 << 28))
+/bin/runtime-workload-elf /srv/workload
+EOF
```

Comment:
What is line 2 to 8 doing?

Reply:
It's just creating the initial bash script that runs from the init-rc script, rather than having a file in the repo and copying it during image build.

Reply:
Otherwise, it's copied from the `nodejs` and `python` runtimes: setting up the network, faking randomness, delaying using factorial, and finally invoking the runtime binary (`runtime-workload-elf`).